### ISSUE TITLE
Missing chapter which is available in the source site and which I got a notification from.

### ISSUE THREAD

**Issue by Jamerrone** [Aidoku/Aidoku#511]
So I got a notification that chapter 198 of 100 girlfriends was released and I can see it in the source website but the chapter itself is missing. I have removed and re-added the manga to the library but that did not fix it. Not sure if this is a source issue or app issue.

https://github.com/user-attachments/assets/c89969fa-128c-4498-a46a-bbabb30a2079

**Comment by Skittyblock** [comment]
you need to pull to refresh on the left half of the screen not the right. 

not quite sure why you would get an update for it without it listing in the source, unless it had been pulled from the source. I don't know if there's much we can really do though if there aren't any steps to reproduce because I'm not sure how this would happen

**Comment by Jamerrone** [comment]
Yeah... after recording this I noticed that I could pull to refresh on the left side but that did not fix it either. It fixed itself the next day I checked, but something else happened. Some manga lost their badges even though they have new/unread chapters. They were there before and they simply disappeared, this just affects a few series but from different sources. As we can't reproduce this, you are free to close it.
